[FIX] ec2 vpcpeering referencers not working

Dkaykay opened the pull request:
### Description of your changes

Changed code-generator config yaml to correctly exclude VpcId and PeerVpcId
Fixes #1033

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

haarchri commented:
@Dkaykay can you please cleanup your commit history ?

haarchri commented:
@Dkaykay thanks for Contribution i added a new PR with clean history and changed preCreate() and tagger here: https://github.com/crossplane/provider-aws/pull/1035 is it okay for you to close this PR ?

Dkaykay commented:
> @Dkaykay thanks for Contribution i added a new PR with clean history and changed preCreate() and tagger here: https://github.com/crossplane/provider-aws/pull/1035 is it okay for you to close this PR ?

Hey @haarchri, 
sure, sounds good to me!
